src/core/borrower/collect/collector.sol

```diff
@@ -90,11 +91,12 @@ contract Collector is DSNote, Auth {
         shelf.claim(loan, address(this));
     }
 
-    function collect(uint loan, address usr) public auth_collector {
-        require(usr == tags[loan].usr || tags[loan].usr == address(0));
+    function collect(uint loan, address buyer) public auth_collector {
+        require(buyer == prices[loan].buyer || prices[loan].buyer == address(0));
         (address registry, uint nft) = shelf.token(loan);
-        shelf.recover(loan, usr, tags[loan].wad);
-        NFTLike(registry).transferFrom(address(this), usr, nft);
+        require(prices[loan].nftPrice > 0, "no nft price defined");
```

Comment:
didn't we say `contract/no-spaces-in-revert` in the styleguide?

Reply:
fixed